jupyterhub/values.yaml

```diff
@@ -137,6 +137,8 @@ hub:
 
 rbac:
   enabled: true
+  serviceAccount:
+    annotations: {}
```

Comment:
There are more service accounts because we have more pods that all need som permissions against the k8s-apiserver, and I think we would need annotations specific to the service accounts.

So, something related to hub must be part of the config of this annotation, so its not applied to all serviceaccounts

Reply:
I will follow your suggestion. I added it to the `rbac` section because it was it changes files in the `rbac` yml file but I have nothing against moving it to the `hub` section